api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -0,0 +1,2168 @@
+      properties:
+        fsType:
+          description: 'Filesystem type of the volume that you want to mount. Tip:
+            Ensure that the filesystem type is supported by the host operating system.
+            Examples: "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
+            More info: https://kubernetes.io/docs/concepts/storage/volumes#rbd'
+          title: Fstype
+          type: string
+        image:
+          description: 'The rados image name. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: Image
+          type: string
+        keyring:
+          description: 'Keyring is the path to key ring for RBDUser. Default is /etc/ceph/keyring.
+            More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: Keyring
+          type: string
+        monitors:
+          description: 'A collection of Ceph monitors. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          items:
+            type: string
+          title: Monitors
+          type: array
+        pool:
+          description: 'The rados pool name. Default is rbd. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: Pool
+          type: string
+        readOnly:
+          description: 'ReadOnly here will force the ReadOnly setting in VolumeMounts.
+            Defaults to false. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: Readonly
+          type: boolean
+        secretRef:
+          allOf:
+          - $ref: '#/components/schemas/LocalObjectReference'
+          description: 'SecretRef is name of the authentication secret for RBDUser.
+            If provided overrides keyring. Default is nil. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: Secretref
+        user:
+          description: 'The rados user name. Default is admin. More info: https://releases.k8s.io/HEAD/examples/volumes/rbd/README.md#how-to-use-it'
+          title: User
+          type: string
+      required:
+      - image
+      - monitors
+      title: RBDVolumeSource
+      type: object
+    ResourceFieldSelector:
+      properties:
+        containerName:
+          description: 'Container name: required for volumes, optional for env vars'
+          title: Containername
+          type: string
+        divisor:
+          allOf:
+          - $ref: '#/components/schemas/Quantity'
+          description: Specifies the output format of the exposed resources, defaults
+            to "1"
+          title: Divisor
+        resource:
+          description: 'Required: resource to select'
+          title: Resource
+          type: string
+      required:
+      - resource
+      title: ResourceFieldSelector
+      type: object
+    RestartPolicy:
```

Comment:
We should remove this because we don't actually support this, right?

Reply:
Done, thanks!